### PR TITLE
RELATED: RAIL-3921 Add notice for react-scripts 5 not working currently

### PR DESCRIPTION
**Pull request by no23reason**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/914/ : SUCCESS in 2m 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/607/ : SUCCESS in 20s

File: docs/02_start__no_boilerplate.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ id: platform_integration
 This document outlines the important steps you need to undertake if you need to integrate GoodData.UI into an existing
 React application or for some reason you cannot use [accelerator toolkit](02_start__using_boilerplate.md) to bootstrap a new application.
 
+## Before you begin
+
+If you used `create-react-app` to create your application, please make sure the `react-scripts` dependency is set to version 4.0.3. Versions 5.0.0 and newer are currently not supported by GoodData.UI.
```
Comment (evgeniabogranskaya) [inline docs/02_start__no_boilerplate.md:13]: If you used `create-react-app` to create your application, make sure that the `react-scripts` dependency is set to version 4.0.3. Version 5.0.0 and newer is not supported.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/915/ : SUCCESS in 1m 15s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/608/ : SUCCESS in 8s

**Review by kandl — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/916/ : SUCCESS in 1m 26s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/175/ : SUCCESS in 58s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/70/ : SUCCESS in 48s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/265/ : SUCCESS in 18s

**Pull request merged**